libs/ptg-ui-react-lib/src/lib/download-file/downloadFile.tsx

```diff
@@ -149,11 +149,6 @@ const PtgUiDownload = ({
 
   return (
     <div className="container">
-      <div className="row">
-        <div className="col-12">
```

Comment:
why this change is needed ?

Reply:
It is common component so i cannot place the toggle button here, so i have removed File Download heading from and added that heading with toggle button react-app download.tsx file